configuration/blockly/rules-blockly-run-and-process.md

```diff
@@ -81,9 +81,22 @@ _Function:_ Retrieve event context related information of the rule
 
 A rule carries contextual information when triggered - this block can retrieve that information.
 
-- The rule trigger type defines which of the attributes are available to the rule - they will not all be available.
+- The rule trigger type defines which of the attributes are available to the rule - not not all be available.
```

Comment:
Not sure what was intended here, but it seems something went wrong ("not not all").